defs.bzl

```diff
@@ -21,10 +21,12 @@ load("//internal/json_to_js:defs.bzl", _json_to_js = "json_to_js")
 load("//internal/toml_to_js:defs.bzl", _toml_to_js = "toml_to_js")
 load("//internal/vue_component:defs.bzl", _vue_component = "vue_component")
 load("//internal/nodejs_jest_test:defs.bzl", _node_jest_test_macro = "node_jest_test_macro")
-load("//internal:node_contrib_repositories.bzl", _node_contrib_repositories = "node_contrib_repositories")
+load("//internal/js_library:defs.bzl", js_library = "node_jest_test_macro")
```

Comment:
why the `js_library = "node_jest_test_macro"`? am I misreading, or what does jest have to do with this change?